app/uk/gov/hmrc/exports/models/declaration/Locations.scala

```diff
@@ -30,7 +30,7 @@ object GoodsLocation {
   implicit val format: OFormat[GoodsLocation] = Json.format[GoodsLocation]
 }
 
-case class OfficeOfExit(officeId: Option[String], presentationOfficeId: Option[String], circumstancesCode: Option[String])
+case class OfficeOfExit(officeId: Option[String], presentationOfficeId: Option[String], circumstancesCode: Option[String], answer: Option[String])
 
```

Comment:
Suggest renaming this field to indicate what it is the answer to.

Reply:
Good spot.
Hopefully is more meaningful now.